src/amo/components/Header/styles.scss

```diff
@@ -1,7 +1,7 @@
 @import '~amo/css/styles';
 
 .Header {
-  background: $ink-90;
+  background: $color-ink-80;
```

Comment:
This needs to be hidden behind the `enableFeatureHeroRecommendation` config flag.